### PR TITLE
coming soon page added

### PR DESCRIPTION
**Pull request by Anupam-dagar**
Live link:
http://anupamdagar.me/hack-in-the-north-preview-repo

### CODE REVIEW

File: comingsoon/index.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        <!-- HERO SECTION -->
+        <div class="row min-full-height"
+        id="hero">
+        <div class="col-10 offset-1">
+
+            <!-- The navbar is contained within a row with id="nav-bar" -->
+            <div class="hidden-md-down row text-center text-white" id="nav-bar">
+                <div class="col-2 offset-1"><a href="https://www.hackinthenorth.com/2016/"><h4 class="nav-bar-link">HINT 2016</h4></a></div>
+                <div class="col-2"><a href="https://www.hackinthenorth.com/2017/"><h4 class="nav-bar-link">HINT 2017</h4></a></div>
+                <div class="col-2"><img src="assets/logo.png" class="nav-bar-logo"></div>
```
Comment (hackerkid) [inline comingsoon/index.html]: This logo looks redundant.

File: comingsoon/index.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                <div class="col-2"><a href="https://www.hackinthenorth.com/team.html"><h4 class="nav-bar-link">TEAM</h4></a></div>
+                <div class="col-2"><a href="https://www.hackinthenorth.com"><h4 class="nav-bar-link">v3.0</h4></a></div>
```
Comment (hackerkid) [inline comingsoon/index.html]: What is the purpose of this link? Can we have something useful with this?

File: comingsoon/index.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+                        <span class="circle bottom-right"></span>
+                        <span class="circle bottom-right"></span>
+                        <span class="circle bottom-right"></span>
+                    </span>
+                </span>
+            </div>
+        </div>
+        <div class="col-6 hackathon">
+            <h1 style="font-family: 'Montserrat', sans-serif;">HACK IN THE NORTH</h1>
```
Comment (hackerkid) [inline comingsoon/index.html]: Should probably specify 3.0 or 2017

**Comment by hackerkid** [comment]
This looks great @Anupam-dagar ! Posted some comments.

File: comingsoon/index.html [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    <div class="container-fluid">
+
+        <!-- HERO SECTION -->
+        <div class="row min-full-height"
+        id="hero">
+        <div class="col-10 offset-1">
+
+            <!-- The navbar is contained within a row with id="nav-bar" -->
+            <div class="hidden-md-down row text-center text-white" id="nav-bar">
+                <div class="col-2 offset-1"><a href="https://www.hackinthenorth.com/2016/"><h4 class="nav-bar-link">HINT 2016</h4></a></div>
```
Comment (hackerkid) [inline comingsoon/index.html]: We should stick with either HINT <year> or HINT <version> through out the web page.

File: comingsoon/index.html (hunk 1 above)
Comment (Anupam-dagar) [reply comingsoon/index.html]: Removing this logo will make the nav bar look plain.I think replacing logo in footer with other data or IIITA logo can make it better.

File: comingsoon/index.html (hunk 2 above)
Comment (Anupam-dagar) [reply comingsoon/index.html]: HINT 2017 link can be replaced with its project showcase one.
Same can be done with 2016 one and v3.0 with the gallery.

File: comingsoon/index.html (hunk 4 above)
Comment (Anupam-dagar) [reply comingsoon/index.html]: Okay, I'll stick with version throughout the page.

**Comment by Anupam-dagar** [comment]
@gauthamzz @yasharmaster it's now responsive.Please check for bugs if there are any on your screens.I need help in fixing overlay issue, having a hard time fixing it.

**Comment by gauthamzz** [comment]
thanks @Anupam-dagar

**Comment by Anupam-dagar** [comment]
You're welcome :smile:

**Comment by mubaris** [comment]
HINT v2.0 link is redirecting to 2016 website and HINT v3.0 is redirecting to 2017 website. Links should be HINT v1.0 and HINT v2.0

**Comment by Anupam-dagar** [comment]
Noted, fixing it.

On Sep 3, 2017 11:14, "Mubaris NK" <notifications@github.com> wrote:

HINT v2.0 link is redirecting to 2016 website and HINT v3.0 is redirecting
to 2017 website. Links should be HINT v1.0 and HINT v2.0

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/HackInTheNorth/hackinthenorth.github.io/pull/117#issuecomment-326785797>,
or mute the thread

**Pull request merged**